### DISCUSSION TITLE
quarkus-hibernate-search-orm-elasticsearch

### DISCUSSION THREAD

**Discussion by labdidi** [quarkusio/quarkus#35408] [Q&A] [answered]
Is there a reactive extension of "quarkus hibernate search elasticsearch" like the non reactive one "quarkus-hibernate-search-orm-elasticsearch"?
In a reactive use like:

`quarkus create app reactive-demo -x hibernate-reactive-panache,reactive-pg-client,resteasy-reactive,resteasy-reactive-jackson --no-code`

it is not so easy to use hibernate search and elasticsearch just with annotations together like the non reactive variant.

[a comment by an automated account is not shown]

**Comment by yrodiere** [comment]
There is no such extension at the moment, because Hibernate Search itself lacks an integration to Hibernate Reactive at the moment, and it's not being actively worked on, either. See here if you want to get involved: https://hibernate.atlassian.net/browse/HSEARCH-4922

**Reply by gsmet** [reply, marked as answer]
Also note that it's perfectly fine to use Hibernate ORM with RESTEasy Reactive. RESTEasy Reactive supports blocking workloads. And it's not obvious that it will not perform as well as the same stuff with Hibernate Reactive.

**Reply by labdidi** [reply]
Thank for the answer!
